development/tools/cmd/dnscollector/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/kyma-project/test-infra/development/tools/pkg/common"
+	"github.com/kyma-project/test-infra/development/tools/pkg/dnscollector"
+	log "github.com/sirupsen/logrus"
+	"golang.org/x/oauth2/google"
+	compute "google.golang.org/api/compute/v1"
+	dns "google.golang.org/api/dns/v1"
+)
+
+const defaultAddressRegexpList = "(remoteenvs-)?gkeint-(pr|commit)-.*,(remoteenvs-)?gke-upgrade-(pr|commit)-.*"
```

Comment:
"Remote Environments" - I think you should use the term "Application" to be consistent with the naming scheme used in Kyma.